File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/prompt_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PromptVO:
    """AI提示语响应"""
    created_at: str = None
    updated_at: str = None
    id: int = None
    title: str = None
    content: str = None
    type: str = None
    biz_type: str = None
    description: str = None
    cate_id: int = None
    enabled: bool = None
    sort: int = None
    parameters: Dict[str, Any] = None
    creator: str = None
    model: str = None
    tags: TagsContent = None
    usage_count: int = None
    avg_response_time: int = None
    version: str = None
    is_public: bool = None
    is_favorite: bool = None
    favorite_count: int = None
    last_used_at: str = None

File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/PromptVO.kt
package com.sdkwork.app

data class PromptVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val id: Int? = null,
    val title: String? = null,
    val content: String? = null,
    val type: String? = null,
    val bizType: String? = null,
    val description: String? = null,
    val cateId: Int? = null,
    val enabled: Boolean? = null,
    val sort: Int? = null,
    val parameters: Map<String, Any>? = null,
    val creator: String? = null,
    val model: String? = null,
    val tags: TagsContent? = null,
    val usageCount: Int? = null,
    val avgResponseTime: Int? = null,
    val version: String? = null,
    val isPublic: Boolean? = null,
    val isFavorite: Boolean? = null,
    val favoriteCount: Int? = null,
    val lastUsedAt: String? = null
)
